model/auth_config.go

```diff
@@ -11,4 +11,17 @@ type AuthConfig struct {
 	AllowedIdentities []client.Identity `json:"allowedIdentities"`
 	GithubConfig      GithubConfig      `json:"githubConfig"`
 	ShibbolethConfig  ShibbolethConfig  `json:"shibbolethConfig"`
+	LdapConfig        LdapConfig        `json:"ldapConfig"`
 }
+
+type TestAuthConfig struct {
+	AuthConfig AuthConfig `json:"authConfig"`
+	Code       string     `json:"code"`
+}
+
+const (
+	AccessModeSetting        = "api.auth.access.mode"
+	AllowedIdentitiesSetting = "api.auth.allowed.identities"
+)
+
+var RancherClient *client.RancherClient
```

Comment:
@prachidamle These settings are moved to `model` package because ldap_provider needs to use them to get the values from db, and an import cycle was getting created by leaving them in `server` package